doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -86,6 +86,8 @@ Mobile targets are currently using a limited version of XAML Hot Reload and do n
 
 Hot Reload is supported by Visual Studio for WinAppSDK and provides support in unpackaged deployment mode.
 
+---
+
 ## Hot Reload Indicator
```

Comment:
@rajamatt I am noticing while testing the documentation page that this image seems a bit blurry compared to the other images. Can you update it for a better quality image if possible please
![image](https://github.com/unoplatform/uno/assets/16295702/aedf58e5-4f9d-48f5-8994-b45351239d5b)